boris-http/boris-http.cabal

```diff
@@ -111,6 +112,7 @@ executable boris-http
                     , ambiata-boris-http
                     , ambiata-boris-queue
                     , ambiata-mismi-core
+                    , ambiata-mismi-dynamodb
```

Comment:
Where is this being pulled in?

Reply:
> Where is this being pulled in?

https://github.com/markhibberd/boris/blob/master/boris-http/main/boris-http.hs#L11